org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/parser/Parser.java

```diff
@@ -57,7 +56,6 @@
 import org.eclipse.jdt.internal.compiler.ASTVisitor;
 import org.eclipse.jdt.internal.compiler.CompilationResult;
 import org.eclipse.jdt.internal.compiler.ast.*;
-import org.eclipse.jdt.internal.compiler.ast.StringTemplate;
```

Comment:
Someone steals imports they just recently added (for good reason).

Reply:
Probably because BREE is not set to 21, but 17 where it does not matter. Consider mentioning in https://github.com/eclipse-platform/.github/discussions/188